.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 
 rvm:
-  - 2.2.3
+  - 2.2.5
```

Comment:
Not sure about `2.2`. I had tried it and the install failed.

Determined it's a dependency by looking at `Gemfile.lock`. Could see it's being included by `listen`, which is required by `guard`.

Not sure if you all use `guard` still, maybe can be removed.